.travis.yml

```diff
@@ -0,0 +1,52 @@
+sudo: false
+language: python
+
+matrix:
+  include:
+    # Use the built in venv for linux builds
+    - os: linux
+      sudo: false
+      python: "2.7"
+    - os: linux
+      sudo: false
+      python: "3.4"
+    - os: linux
+      sudo: false
+      python: "3.5"
+    - os: linux
+      sudo: false
+      python: "3.6"
+    # Travis doesn't have Python 3.7 activated yet
+    # - os: linux
+    #   sudo: false
+    #   python: "3.7"
+
+    # Use generic language for osx
+    - os: osx
+      language: generic
+      env: PYTHON_VERSION=2.7
+    - os: osx
+      language: generic
+      env: PYTHON_VERSION=3.4
+    - os: osx
+      language: generic
+      env: PYTHON_VERSION=3.5
+    - os: osx
+      language: generic
+      env: PYTHON_VERSION=3.6
+    - os: osx
+      language: generic
+      env: PYTHON_VERSION=3.7
+
+before_install:
+  - .travis/install.sh
+
+before_script:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then source $HOME/miniconda/bin/activate venv; fi
+  - pip install --upgrade setuptools
+  - pip install pathtools3 monotonic pytest pytest-cov pytest-timeout coveralls
```

Comment:
I think this line is not required because dependecies will automatically be installed when calling `python setup.py test`.